appveyor.yml

```diff
@@ -12,7 +12,7 @@ cache:
 
 environment:
   global:
-    nodejs_version: 6
+    nodejs_version: 10
```

Comment:
Looks like this modifies Appveyor to _only_ test under node10? Was that the intention?! :-/ https://www.appveyor.com/docs/lang/nodejs-iojs/#testing-under-multiple-versions-of-nodejs-or-iojs shows how to test multiple versions...

Reply:
Yes, and no – testing something (and forgot to mark this WIP) :)